DDCore/include/XML/UnicodeValues.h

```diff
@@ -192,9 +192,11 @@ UNICODE (global);
 UNICODE (global_grid_xy);
 UNICODE (grid_size_x);
 UNICODE (grid_size_y);
-UNICODE (grid_size_z);
+UNICODE (grid_size_u);
```

Comment:
```suggestion
UNICODE (grid_size_z);
UNICODE (grid_size_u);
```

Reply:
You had inadvertently removed `grid_size_z`. I wanted to point out it should be added back.